api/scripts/opensea-events.py

```diff
@@ -0,0 +1,76 @@
+import asyncio
+import logging
+import os
+import sys
+import time
+
+
+from core.requester import Requester
+from core.util import date_util
+from core.exceptions import NotFoundException
+from core.store.database import Database
+
+
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
+from notd.token_listing_processor import TokenListingProcessor
+from notd.model import COLLECTION_SPRITE_CLUB_ADDRESS, COLLECTION_GOBLINTOWN_ADDRESS
+from notd.store.retriever import Retriever
+from notd.store.saver import Saver
+openseaApiKey = os.environ['OPENSEA_API_KEY']
+
+
+async def test():
+    startHour = date_util.datetime_from_string('2022-07-26T23:20:00.000000')
+    endHour = date_util.datetime_from_datetime(dt=startHour, minutes=60)
+    databaseConnectionString = Database.create_psql_connection_string(username=os.environ["DB_USERNAME"], password=os.environ["DB_PASSWORD"], host=os.environ["DB_HOST"], port=os.environ["DB_PORT"], name=os.environ["DB_NAME"])
+    database = Database(connectionString=databaseConnectionString)
+    retriever = Retriever(database=database)
+    saver = Saver(database=database)
+    requester = Requester()
+    openseaRequester = Requester(headers={"Accept": "application/json", "X-API-KEY": openseaApiKey})
+    tokenListingProcessor = TokenListingProcessor(requester=requester, openseaRequester=openseaRequester)
+    registryAddress = COLLECTION_GOBLINTOWN_ADDRESS
+    queryData = {
+        'asset_contract_address': registryAddress,
+        "occurred_after": startHour,
+    }
+    await database.connect()
+    while startHour< endHour:
+        #print(startHour, 'after')
+        tokensToReprocess = []
+        queryData['occurred_after'] = startHour
+        queryData['occurred_before'] = date_util.datetime_from_datetime(dt=startHour, minutes=5)
+        response = await openseaRequester.get(url="https://api.opensea.io/api/v1/events", dataDict=queryData)
+        responseJson = response.json()
+        for asset in responseJson['asset_events']:
```

Comment:
is there not a limit to how many get returned here? I'm sure you will need to do some paging based on the response

Reply:
ping on this